mystbin/frontend/Dockerfile

```diff
@@ -1,11 +1,24 @@
-FROM node:latest
+FROM node:16-alpine as dep
 
-RUN mkdir -p /app
+# avoid rebuilding the node modules on every single file change, instead only when our deps change.
 WORKDIR /app
+COPY package.json yarn.lock ./
+RUN yarn install --frozen-lockfile
 
-COPY . /app
+FROM node:16-alpine AS builder
 
-RUN yarn install --production
+ENV NODE_ENV=production
+WORKDIR /app
+COPY . .
+COPY --from=dep /app/node_modules ./node_modules
+RUN yarn build
+
+FROM node:16-alpine AS runner
+
+ENV NODE_ENV=production
+COPY --from=builder /app/package.json ./
+COPY --from=builder /app/public ./public
+COPY --from=builder /app/.next ./.next
+COPY --from=builder /app/node_modules ./node_modules
 
-ENV NODE_OPTIONS="--openssl-legacy-provider"
-CMD ["yarn", "run", "launch"]
+CMD [ "yarn", "run", "start"]
```

Comment:
I believe we run with these config options due to something broken in the frontend deps. Can you confirm this works before I merge the PR?
The `NODE_OPTIONS` historically resolved a bug for us.